scripts/merge_geojson/merge_geojson.py

```diff
@@ -0,0 +1,210 @@
+        del properties["fix_suggestions"]
+    return feature_collection
+
+
+def write_feature_rfc7464(file_name: str, feature: geojson.Feature):
+    """
+    Create an RFC7464 compliant file for use with MapRoulette
+    """
+    fc = (
+        fix_suggestions_to_cooperativeWork(feature)
+        if "fix_suggestions" in feature["properties"]
+        else geojson.FeatureCollection([feature])
+    )
+    with open(file_name, "ab") as fh:
+        # Line-by-line uses the 1E byte record separator
+        fh.write(ASCII_RECORD_SEPARATOR)
+
+    with open(file_name, "a") as fh:
+        string = geojson.dumps(fc)
+        fh.write(string)
+
+    with open(file_name, "ab") as fh:
+        # Specific line separator (byte used instead of specific character,
+        # just in case)
+        fh.write(ASCII_LINE_SEPARATOR)
+
+
+def read_file(file_name: str) -> geojson.FeatureCollection:
+    file_type = "gz" if file_name.endswith(".gz") else "geojson"
+    try:
+        if file_type == "geojson":
+            with open(file_name, "r") as fh:
+                first = str.encode(fh.read(1))
+                if first == ASCII_RECORD_SEPARATOR:
+                    return None
+                fh.seek(0)
+                return geojson.load(fh)
+        elif file_type == "gz":
+            with gzip.open(file_name, "rt") as fh:
+                return geojson.load(fh)
+    except json.decoder.JSONDecodeError as e:
+        logging.error(f"{file_name} had a json parse error")
+        raise e
+
+    raise ValueError(f"{file_name} is not a recognized type")
+
+
+def line_by_line(files: typing.List[str], check: str):
+    write_name = check + ".line.geojson"
+    if len(files) > 0:
+        write_name = os.path.join(os.path.dirname(files[0]), write_name)
+    if os.path.isfile(write_name):
+        os.remove(write_name)
+    for f in files:
+        fc = read_file(f)
+        if fc is None:
+            continue
+        if isinstance(fc, geojson.FeatureCollection):
+            for feature in fc["features"]:
+                write_feature_rfc7464(write_name, feature)
+        else:
+            logging.error(f"Bad file? {f}")
+
+
+def concat_files(files: typing.List[str], check: str):
+    write_name = check + ".concat.geojson"
+    if len(files) > 0:
+        write_name = os.path.join(os.path.dirname(files[0]), write_name)
+    featurecollection = None
+    for f in files:
+        fc = read_file(f)
+        if fc is None:
+            continue
+        if featurecollection is None:
+            featurecollection = fc
+        else:
+            featurecollection["features"] = (
+                featurecollection["features"] + fc["features"]
+            )
+    with open(write_name, "w") as fp:
+        geojson.dump(featurecollection, fp)
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Create a line-by-line geojson file"
+    )
+    parser.add_argument("files", nargs="+")
+    parser.add_argument("-c", "--checks", nargs="?")
```

Comment:
Based on the use of this argument it appears to be expecting a single check name. So I think it would make more sense to have the flag be --check.